Source/EmbeddedBoundary/WarpXInitEB.cpp

```diff
@@ -1,4 +1,5 @@
 #include "WarpX.H"
+#include <cmath>
```

Comment:
please include stdlib headers last, so the code is more robust when internal includes get updated
https://warpx.readthedocs.io/en/latest/developers/repo_organization.html#c-includes

Reply:
Ok, thanks I'll keep that in mind also for the future. Actually this include was outdated so I removed it.